app/controllers/images_controller.rb

```diff
@@ -1,5 +1,7 @@
 class ImagesController < ApplicationController
-  def index; end
+  def index
+    @images = Image.all.reverse
+  end
```

Comment:
Adding them in reverse because I want the newest image to be shown first on the view